src/SparsityPatternView.hpp

```diff
@@ -224,16 +229,19 @@ class SparsityPatternView : protected ArrayOfSetsView< COL_TYPE, INDEX_TYPE >
     return ParentClass::insertIntoSet( row, col );
   }
 
+  /// @cond DO_NOT_DOCUMENT
```

Comment:
@rrsettgast If I change the name of the method it works just fine, as is doxygen doesn't recognize the parameters.